Sources/XMTPiOS/Frames/FramesClient.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  File.swift
+//  
+//
+//  Created by Alex Risch on 3/28/24.
+//
+
+import Foundation
+import LibXMTP
+
+public typealias FrameActionBody = Xmtp_MessageContents_FrameActionBody
+public typealias FrameAction = Xmtp_MessageContents_FrameAction
+
+class FramesClient {
+    var xmtpClient: Client
+    var proxy: OpenFramesProxy
+
+    init(xmtpClient: Client, proxy: OpenFramesProxy? = nil) {
```

Comment:
Same for the init if you want people to be able to access it.
```suggestion
    public init(xmtpClient: Client, proxy: OpenFramesProxy? = nil) {
```